lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -3161,11 +3161,13 @@ void HdVP2Material::CompiledNetwork::_UpdateShaderInstance(
                 if (isMaterialXNode) {
                     const MString paramName = "_" + nodeName + "file_sampler";
                     samplerStatus = _surfaceShader->setParameter(paramName, *sampler);
+                    updatedAttributes.insert(paramName);
```

Comment:
PR #3043 modifies all preceding lines, so automatic merge conflicts are to be expected.